react-native/react/engine/index.js

```diff
@@ -92,6 +94,24 @@ class Engine {
     this.generalListeners[method] = (this.generalListeners[method] || []).filter(l => l !== listener)
   }
 
+  listenServerInit (method, listener) {
+    this.serverListeners[method] = listener
+  }
+
+  _serverInitIncomingRPC (method, param, response) {
+    const sid = this.getSessionID()
+    const cbs = {
+      start: callMap => {
+        this.sessionIDToIncomingCall[sid] = callMap
+        response.result(sid)
+      },
+      end: () => {
+        delete this.sessionIDToIncomingCall[sid]
+      }
+    }
+    this.serverListeners[method](param, cbs)
```

Comment:
Turns out this code is only called if there is a `this.serverListeners[method]`. See: [here](https://github.com/keybase/client/pull/1318/files#diff-9faed3ed3de893e2f4a99f81e783574eR178)

If there isn't it'll print the existing error of `Unknown incoming RPC`